jakarta/transform.sh

```diff
@@ -237,9 +279,58 @@ build_module_only_no_tests "extensions/vertx-http"
 build_module "extensions/vertx-http/dev-console-runtime-spi"
 build_module "extensions/vertx-http/dev-console-spi"
 build_module "extensions/arc"
+build_module "extensions/smallrye-context-propagation"
+build_module "extensions/mutiny"
+build_module "extensions/netty"
+build_module "extensions/vertx"
+build_module "extensions/security"
+build_module_only_no_tests "extensions/kubernetes"
+build_module "extensions/kubernetes/spi"
+build_module "extensions/vertx-http"
+build_module "extensions/jsonp"
+build_module "extensions/jaxrs-spi"
+build_module "extensions/undertow"
+build_module "extensions/jsonb"
+build_module "extensions/jackson"
+build_module "extensions/jaxp"
+build_module "extensions/jaxb"
+build_module "extensions/apache-httpclient"
+
+# this will be simplified once we can get all RESTEasy Classic to compile fine
+build_module_only_no_tests "extensions/resteasy-classic"
+build_module "extensions/resteasy-classic/resteasy-common"
+build_module "extensions/resteasy-classic/resteasy-server-common"
+build_module "extensions/resteasy-classic/resteasy"
+build_module "extensions/resteasy-classic/resteasy-jsonb"
+build_module "extensions/resteasy-classic/resteasy-jackson"
+build_module "extensions/resteasy-classic/resteasy-jaxb"
+build_module "extensions/resteasy-classic/resteasy-links"
+build_module "extensions/resteasy-classic/resteasy-mutiny-common"
+build_module "extensions/resteasy-classic/resteasy-mutiny"
```

Comment:
I will simplify this once I can get the client part to work but for now it will do.